server/validation/validation.ts

```diff
@@ -29,6 +30,22 @@ Validator.register(
   'Enter a real date, like 2020-03-23',
 )
 
+Validator.register(
+  'after',
+  (date: string, params: string): boolean => {
+    const val1 = date
+    const val2 = params.split(',')[0]
+
+    if (!isISO8601(val1, { strict: true }) || !isISO8601(val2, { strict: true })) return false
+
+    const inputDate = moment(val1, 'YYYY-MM-DD')
+    const afterDate = moment(val2, 'YYYY-MM-DD')
+
+    return inputDate.isAfter(afterDate)
+  },
+  'The :attribute must be after :after.',
+)
+
```

Comment:
The after validation in validatorjs is completely broken and the issue for it is years old. So had to create our own implementation.